### PR TITLE
docs: deprecate cwa-risk-assessment.md

### PR DESCRIPTION
**Pull request by mlenkeit**
- add deprecation notice to document
- add reference to solution architecture

related to #453

**Background:**

We realistically don't have enough time at the moment to rework the document. At the same time, there is a pretty detailed description of the risk assessment and risk calculation in the Solution Architecture, which reflects what CWA is currently doing.

Instead of completely deleting the file, we propose to add a deprecation notice instead and point to the respective section in the Solution Architecture.

### CODE REVIEW

**Comment by mlenkeit** [comment]
/fyi @MikeMcC399 will leave the PR open for a few days, happy to get your feedback if you have some time 😊

**Comment by Ein-Tim** [comment]
(I know I wasn't asked but still): Looks good to me & makes sense if there is no plan to update this document in the near future.

**Review by GisoSchroederSAP — APPROVED** [review]
The deprecation notice is clear and refers to an alternative source of information.
This is absolutely fine.

**Comment by MikeMcC399** [comment]
@mlenkeit 

I'm glad to give feedback. Thank you for asking! I support the proposal to deprecate the document and leave it online if it cannot be updated.

The statement in the PR contents that the document is accurate for versions < 1.8 is not entirely correct because it doesn't cover the use of Onset of Symptoms which was introduced in 1.5. We can more accurately say that the document covers versions prior to 1.5.

The section "Later calculation improvements" was something which I authored and I suggest this should be removed completely if the document is to be deprecated. There is now no need to try to describe the increments of functionality (although the FAQs do document these changes). Instead the reference to the [Solution Architecture](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md) document is sufficient to understand how the risk calculation is carried out since the migration to ENF V2 / ENS ExposureWindow mode in app version 1.9 on Dec 16, 2020 (see blog post [Corona-Warn-App Version 1.9 to use version 2 of the Exposure Notification Framework](https://www.coronawarn.app/en/blog/2020-12-16-corona-warn-app-version-1-9/)).

The deprecation also affects the German version of the document https://github.com/corona-warn-app/cwa-documentation/blob/main/translations/cwa-risk-assessment.de.md which needs similar changes making to it.

In addition, references to the English and German versions of the document in the respective README files need to be updated. They can point to the [Solution Architecture > Mobile Application](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md#mobile-applications) section instead.

If you would like a PR to cover the above, I can submit one on top of your changes. Please let me know.

---

The document [cwa-risk-assessment.md](https://github.com/corona-warn-app/cwa-documentation/blob/main/cwa-risk-assessment.md) is mentioned on the cwa-website in the FAQ https://www.coronawarn.app/en/faq/#encounter_but_green "An encounter has been reported, but the risk status stays green". This FAQ is linked to in the app. 

- I have submitted a separate website issue for the website changes https://github.com/corona-warn-app/cwa-website/issues/2388. 

This means that German language app users with a green encounter will ultimately be referred to an English language document. This is however still preferable to them being referred to an out-of-date German language document. The FAQ link is at a third level of depth: "Niedriges Risiko" Kachel > "Ihr Risikostatus" > [FAQ: "Ich habe eine Risiko-Begegnung, mir wird aber nach wie vor ein niedriges Risiko angezeigt"](https://www.coronawarn.app/de/faq/#encounter_but_green) > https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/cwa-risk-assessment.de.md.

---

The [Solution Architecture > Mobile Application](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md#mobile-applications) section doesn't include some information from the presentation [Behind the scenes of the Corona-Warn-App](https://www.coronawarn.app/en/blog/2020-12-30-cwa-behind-the-scenes/) regarding risk calculation. For instance, I could only find the Transmission Risk Level table in the [presentation document](https://www.coronawarn.app/assets/documents/cwa-rc3.pdf) on page 15. I couldn't see it in the [Solution Architecture > Mobile Application](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md#mobile-applications) document section, nor by referring to any of the links in the document. This should be a separate topic too. So I will submit a new issue about this.

**Comment by MikeMcC399** [comment]
- The following FAQs were fixed to point to the  [Solution Architecture > Mobile Application](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md#mobile-applications) document section through PR https://github.com/corona-warn-app/cwa-website/pull/2389 instead of referring to the now deprecated document:
    - https://www.coronawarn.app/en/faq/#encounter_but_green and 
    - https://www.coronawarn.app/de/faq/#encounter_but_green

**Comment by MikeMcC399** [comment]
- https://github.com/corona-warn-app/cwa-documentation/issues/832 opened to suggest improvements to the  [Solution Architecture](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md) document.

**Comment by MikeMcC399** [comment]
@mlenkeit 
- I submitted #835 to update the README files with links to the Solution Architecture document.

**Comment by MikeMcC399** [comment]
@mlenkeit 

I also submitted PR #836 for you to look at. If you approve, we could merge #836 into your PR here (#829) and then into the `main` branch.

File: cwa-risk-assessment.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # How does the Corona-Warn-App identify an increased risk?
 
----
-**Later calculation improvements**
+Please note that this document is _not_ up-to-date anymore. It reflects the original risk assessment for the Corona-Warn-App before version 1.5 (released in October 2020) and is kept here for historical reference only. An up-to-date description of risk assessment and risk calculation in the Corona-Warn-App is available in the [Solution Architecture > Mobile Application](./solution_architecture.md#mobile-applications) document section.
+
+<br /><br />
+
+## DEPRECATION NOTICE
```
Comment (Ein-Tim) [inline cwa-risk-assessment.md:7]: Should there be an "IMPORTANT!" here, to match the German translation?

Comment (mlenkeit) [reply cwa-risk-assessment.md]: @Ein-Tim see https://github.com/corona-warn-app/cwa-documentation/pull/829/commits/e7997365a93c8f241a9eb689e5421ad3217b908c

**Comment by MikeMcC399** [comment]
@mlenkeit 

It's looking good now! Thanks for the collaboration!

There is just one very minor point, which I hadn't noticed before: The word `-berechnung` may be broken after the hyphen. The simplest solution would be to write out the word in full i.e. Risikoberechnung. I believe there is a character "non-breaking hyphen" but I didn't try this out as an alternative.

image

File: translations/cwa-risk-assessment.de.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 # Wie ermittelt die Corona-Warn-App ein erhöhtes Risiko?
 
----
-**Nachträgliche Verbesserungen der Risikoberechnung**
+Dieses Dokument ist **nicht mehr aktuell**. Es beschreibt die ursprüngliche Risikoberechnung in der Corona-Warn-App vor Version 1.5 (in Oktober 2020 freigegeben) und wird hier aus Dokumentationsgründen noch beibehalten. Eine aktuelle Beschreibung der Risikoanalyse und -berechnung in der Corona-Warn-App befindet sich im Dokumentenabschnitt [Solution Architecture > Mobile Application](../solution_architecture.md#mobile-applications) (auf Englisch).
```
Comment (Ein-Tim) [inline translations/cwa-risk-assessment.de.md:3]:
```suggestion
Dieses Dokument ist **nicht mehr aktuell**. Es beschreibt die ursprüngliche Risikoberechnung in der Corona-Warn-App vor Version 1.5 (im Oktober 2020 veröffentlicht) und wird hier aus Dokumentationsgründen noch beibehalten. Eine aktuelle Beschreibung der Risikoanalyse und -berechnung in der Corona-Warn-App befindet sich im Dokumentenabschnitt [Solution Architecture > Mobile Application](../solution_architecture.md#mobile-applications) (auf Englisch).
```

**Review by Ein-Tim — COMMENTED** [review]
Improve German wording.

**Comment by mlenkeit** [comment]
> The word `-berechnung` may be broken after the hyphen. 

@MikeMcC399  fixed with https://github.com/corona-warn-app/cwa-documentation/pull/829/commits/480567e35101f56453993e01b6b4e7c775cdc06e

**Comment by mlenkeit** [comment]
I think the change is good enough now, thanks @MikeMcC399 and @Ein-Tim for all the contributions. I'll merge the PR. If you have further remarks, feel free to open an issue or a PR.

**Pull request merged**